plugins/orchestrator-common/package.json

```diff
@@ -34,13 +34,15 @@
     "test": "backstage-cli package test --passWithNoTests --coverage",
     "clean": "backstage-cli package clean",
     "prepack": "backstage-cli package prepack",
-    "postpack": "backstage-cli package postpack"
+    "postpack": "backstage-cli package postpack",
+    "generate-models": "openapi-typescript ./api/openapi.yaml -o ./src/auto-generated/api/models/schema.ts"
   },
   "dependencies": {
     "@severlessworkflow/sdk-typescript": "^3.0.3",
     "js-yaml": "^4.1.0",
     "json-schema": "^0.4.0",
-    "openapi-types": "^12.1.3"
+    "openapi-types": "^12.1.3",
+    "openapi-typescript": "^6.7.3"
```

Comment:
`openapi-typescript` can be moved to `devDependencies`

Reply:
Dependency removed